modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoverySpi.java

```diff
@@ -382,7 +386,10 @@
     /** Default SO_LINGER to use for socket. Set negative to disable, non-negative to enable, default is {@DFLT_SO_LINGER }. */
     private int soLinger = DFLT_SO_LINGER;
 
-    /** IP finder clean frequency. */
+    /**
+     * Frequency which coordinator clean IP finder and keeps it in the correct state, unregistering addresses of the
+     * nodes that has left the topology.
+     */
```

Comment:
_The_ frequency _with_ which coordinator clean_s_ IP finder and keeps it in the correct state, unregistering addresses of the
nodes that _have_ left the topology.